Makefile

```diff
@@ -15,3 +15,18 @@ install:
 GOLANGCI_VERSION=v1.27.0
 lint:
 	@docker run --rm -t --user $$(id -u):$$(id -g) -v $$(go env GOCACHE):/cache/go -e GOCACHE=/cache/go -e GOLANGCI_LINT_CACHE=/cache/go -v $$(go env GOPATH)/pkg:/go/pkg -v ${PWD}:/app -w /app golangci/golangci-lint:${GOLANGCI_VERSION}-alpine golangci-lint run ./...
+
+
+release-dev:
+	env=dev make release
+
+release-prd:
+	env=prd make release
+
+PGP_FINGERPRINT=`secrethub read secrethub/logins/${env}/goreleaser/pgp/fingerprint.pgp`
+
+release:
+	secrethub read secrethub/logins/${env}/goreleaser/pgp/private.pgp | gpg --import --batch
+	PGP_FINGERPRINT=${PGP_FINGERPRINT} goreleaser release --rm-dist $(if $(filter ${env},dev),--snapshot,)
+	gpg --yes --batch --delete-secret-key ${PGP_FINGERPRINT}
+	gpg --yes --batch --delete-key ${PGP_FINGERPRINT}
```

Comment:
What happens if you run this without specifying `env`?

Reply:
The current code would deploy to prod. I'll change the condition so that it only deploys to prod if `env` is precisely `"prd"`